test/fixtures/with-errors/.template-lintrc.js

```diff
@@ -1,5 +1,13 @@
 module.exports = {
   rules: {
-    'no-bare-strings': true
-  }
+    'no-bare-strings': true,
+  },
+  overrides: [
```

Comment:
I'd prefer to remove the fixture usage, in favor of the (relatively newish) project helpers landed in #1151. Removing the fixtures should be small "one off" PRs though.